templates/docs/examples/patterns/code-snippet/_script.js

```diff
@@ -0,0 +1,26 @@
+function attachDropdownEvents(dropdown) {
+  dropdown.addEventListener('change', function (e) {
+    var targetElement = document.getElementById(e.target.value);
+
+    togglePanel(targetElement, dropdown.options);
+  });
+}
+
+function togglePanel(targetPanel, options) {
+  for (var i = 0; i < options.length; i++) {
+    var panel = document.getElementById(options[i].value);
+    panel.classList.add('u-hide');
```

Comment:
Should we be toggling aria-hidden as well?

Reply:
Good point